packages/docs-app/src/examples/select-examples/films.tsx

```diff
@@ -108,4 +113,27 @@ export const TOP_100_FILMS = [
     { title: "Monty Python and the Holy Grail", year: 1975 },
 ].map((m, index) => ({ ...m, rank: index + 1 }));
 
-export type Film = typeof TOP_100_FILMS[0];
+export type Film = typeof items[0];
```

Comment:
would rather just make this an interface so its usage is more obvious in the other files.

```ts
interface IFilm {
    ...
}
```